src/ploigos_step_runner/step_implementers/create_container_image/buildah.py

```diff
@@ -115,6 +115,7 @@ def _run_step(self):
         image_spec_file_location = context + '/' + image_spec_file
         application_name = self.get_value('application-name')
         service_name = self.get_value('service-name')
+        tls_verify = self.get_value('tlsverify')
 
```

Comment:
Pretty much everywhere else this property is called `tls-verify`. Can we standardize?

Reply:
Yes! Standardize on `tls-verify`.